addon/globalPlugins/ocr.py

```diff
@@ -243,11 +271,15 @@ def _getPointFromOffset(self, offset):
 		return locationHelper.Point(int(word.left), int(word.top))
 
 
-configspec = {
-	"language": f"string(default={LanguageInfo.fromCurrentNVDALanguage().TesseractLocaleName})"
-}
+configSpecString = ("""
+	language = string(default={defaultLang})
+	quality = string(default=fast)

```

Comment:
Since all possible values for both quality and priorities are known in advance there is no good reason to use strings. Please convert these into a list of `option`.